app/styles/modules/_modal.scss

```diff
@@ -0,0 +1,12 @@
+#modal {
+  background: $white;
+  left: 0;
+  position: absolute;
+  top: 0;
+  z-index: 20;
```

Comment:
ditto for this z-index, although really, we don't have that many layers on screen right now